spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/postgresml/PostgresMlAutoConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package org.springframework.ai.autoconfigure.postgresml;
+
+import org.springframework.ai.embedding.EmbeddingClient;
+import org.springframework.ai.postgresml.PostgresMlEmbeddingClient;
+import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+/**
+ * @author Utkarsh Srivastava
+ */
+@AutoConfiguration
+@ConditionalOnClass(PostgresMlEmbeddingClient.class)
+@EnableConfigurationProperties(PostgresMlProperties.class)
+public class PostgresMlAutoConfiguration {
+
+	@Bean
+	@ConditionalOnMissingBean
+	public EmbeddingClient postgresMlEmbeddingClient(JdbcTemplate jdbcTemplate,
```

Comment:
I think there needs to be some additional configuration to ensure that a `JdbcTemplate` is also created.

Reply:
I think @AutoConfiguration(after = JdbcTemplateAutoConfiguration.class) needs to be added at the class level, i'll add this an merge.